tests/js/spec/components/dashboards/issueWidgetQueriesForm.spec.tsx

```diff
@@ -6,6 +6,7 @@ import {generateFieldOptions} from 'sentry/views/eventsV2/utils';
 
 describe('IssueWidgetQueriesForm', function () {
   const organization = TestStubs.Organization();
+  const api = new MockApiClient();
```

Comment:
should we add a mock for get tags and add a test expecting it to be called?

Reply:
👍  added mock and test!